presentations/index.md

```diff
@@ -1,10 +1,13 @@
+This is a collection of Gluster presentations from all over the world. 
+We have a [slideshare account](http://www.slideshare.net/GlusterCommunity) where most of these presentations are stored. 
+
```

Comment:
@humblec Sure, but maybe someone responsible for the slideshare account can receive the slides, upload them and provide the URL or create a PR for this page?